rootfs/bin/boot

```diff
@@ -78,18 +27,18 @@ else
 fi
 
 echo "Django checks:"
-./manage.py check --deploy api
+./app/manage.py check --deploy api
 
 echo "Health Checks:"
-./manage.py healthchecks
+./app/manage.py healthchecks
 
 echo "Database Migrations:"
-sudo -E -u deis ./manage.py migrate --noinput
+sudo -E -u deis ./app/manage.py migrate --noinput
 
 # spawn a gunicorn server in the background
-sudo -E -u deis gunicorn -c deis/gconf.py deis.wsgi &
+sudo -E -u deis gunicorn -c /app/deis/gconf.py deis.wsgi &
 
-./manage.py load_db_state_to_etcd
+./app/manage.py load_db_state_to_k8s
```

Comment:
Yeah, but `./app/manage.py` isn't absolute. It is relative to your current working directory. If that changes due to `WORKDIR` or some shenanigans upstream you break.